### PR TITLE
Added new codes in folder.

### LINKED ISSUE #22: Coding-Interview-Problems
**I want contribute some famous coding interview problems.**

### PR DESCRIPTION
**Pull request by Shubhamnakawe**
Fixes #22

### CODE REVIEW

File: Coding-Interview-Codes/Patterns/Pymrid_pattern.cpp [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#include<iostream> 
+using namespace std; 
+
```
Comment (APratham) [inline Coding-Interview-Codes/Patterns/Pymrid_pattern.cpp:3]: Change name of file to `Pyramid_pattern.cpp` from `Pymrid_pattern.cpp`

File: Coding-Interview-Codes/Readme.md [hunk 2]
```diff
@@ -78,3 +78,37 @@ Input : arr[] = {20, 10, 20, 4, 100}
...
+
+2. Take the first two terms of the series as public members a and b with values 0 and 1, respectively.
+
+3. Create a generate() method in this class to generate the Fibonacci Series.
+
+4. Create an object of this class and call the generate() method of this class using that object.
+
+5. The Fibonacci Series will get printed.
+
+## [Different Patterns Codes](https://github.com/Shubhamnakawe/BackToBasics-Hacktoberfest/tree/main/Coding-Interview-Codes/Patterns)
```
Comment (APratham) [inline Coding-Interview-Codes/Readme.md:111]: You will have to change this link to our repository not yours. It won't be good if someone clicks on a link and it takes them somewhere entirely different.

File: Coding-Interview-Codes/Readme.md [hunk 3]
```diff
@@ -78,3 +78,37 @@ Input : arr[] = {20, 10, 20, 4, 100}
...
+
+- Triangle Pattern
+- Pymrid Pattern
```
Comment (APratham) [inline Coding-Interview-Codes/Readme.md:114]: Change Pymrid over here too to Pyramid

File: Coding-Interview-Codes/Readme.md (hunk 2 above)
Comment (APratham) [reply Coding-Interview-Codes/Readme.md:111]: You just need to change the username to `MahaWiki` that's all

File: Pyhton/sum_of_array.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
+def _sum(arr,n): 
+
```
Comment (APratham) [inline Pyhton/sum_of_array.py:2]: You will have to change the name of the folder to `Python`

**Review by APratham — CHANGES_REQUESTED** [review]
Kindly go through the changes requested and commit once again

**Comment by Shubhamnakawe** [comment]
I have done the changes. Please check it.

File: Coding-Interview-Codes/Patterns/Pymrid_pattern.cpp (hunk 1 above)
Comment (Shubhamnakawe) [reply Coding-Interview-Codes/Patterns/Pymrid_pattern.cpp]: Changed name to Pyramid_pattern.cpp

**Review by APratham — APPROVED** [review]

**Comment by APratham** [comment]
Awesome. Sorry about the delay

**Pull request merged**